### PR TITLE
Adds new "Help Wanted" indicators to the Repositories and Repository views.

### PR DESCRIPTION
**Pull request by marcesher**
Implements #29.

Expects an integer "HelpWantedIssueCount" field in the JSON API

### CODE REVIEW

**Comment by ondrae** [comment]
@dlapiduz I really like the new version. Surfacing "help wanted" issues has been really useful for us at Code for America.

**Comment by Scotchester** [comment]
@dlapiduz I think it looks great! Love the approach of having an Issues browser.

**Comment by dlapiduz** [comment]
What do you all think about the "Help the United States" banner? I've also thought of "Code for your country" or "Hack the nation" (that looks too extreme)

**Comment by Scotchester** [comment]
Oooh, "Code for your country" is catchy.

Also, I suggest rewording the phrase below that should read "of the hackers, for the hackers, and **by the hackers**". It doesn't match [the original](http://en.wikipedia.org/wiki/Gettysburg_Address) perfectly, but I think it flows better if it starts with "of" like the original and then hammers home the "by" (the concept this website is geared towards) at the end.

**Comment by afeld** [comment]
What happened with this idea?

**Comment by dlapiduz** [comment]
@afeld it is kind of in there but it can be reinforced... I think @marcesher had something in the repo page that I didn't realize it was there...

Maybe we can port it back to the new layout....

**Comment by afeld** [comment]
Oh, derp, totally missed that the heading says "Latest Help Wanted Issues". Wonder if this makes more sense as a filter that's enabled by default?

**Comment by dlapiduz** [comment]
Like this page has a good amount of help wanted issues: https://www.govcode.org/issues but the project pages do not.

**Comment by afeld** [comment]
Or maybe just sort and have help-wanted issues first? Note: not a big deal now that I realize those are being shown first.

**Comment by dlapiduz** [comment]
@afeld I am not sure I follow... the help wanted issues are shown first...

**Comment by afeld** [comment]
Don't mind me :walking:
